src/file-reader/vue-file-reader/vue-file-reader.ts

```diff
@@ -24,21 +24,14 @@ function parseAPI(filePath: string, api: ComponentAPI): Document {
     }
 
     return {
+        kind: "partial",
         id: `fs:${filePath.replace(/\\/g, "/")}`,
         name: `vue:${componentName}`,
         alias: [],
-        visible: false,
-        attributes: { sortorder: Infinity, redirectFrom: [] },
         body: html,
-        outline: [],
         format: "html",
-        tags: [],
-        template: "",
         fileInfo: {
-            path: "",
-            name: "",
-            fullPath: "",
-            outputName: false,
+            fullPath: filePath,
```

Comment:
The big idea is to get rid of most of these properties, most of them which doesn't make sense for a partial document to be inserted into a full document anyway.